src/ts/components/settingspanel.ts

```diff
@@ -91,6 +92,31 @@ export class SettingsPanel extends Container<SettingsPanelConfig> {
     }
   }
 
+  /**
+   * Hack for IE + Firefox
+   * when the settings panel fades out while an item of a select box is still hovered, the select box will not fade out
+   * while the settings panel does. This would leave a floating select box, which is just weird
+   */
+  private hideHoveredSelectBoxes() {
+    this.getItems().forEach((item: SettingsPanelItem) => {
+      if (item.isActive()) {
+        if ((item as any).setting instanceof SelectBox) {
+          const selBox: HTMLElement = (item as any).setting.getDomElement().get()[0];
+          const oldDisplay = selBox.style.display;
```

Comment:
Then use the `DOM` class like everywhere else (the UI components do not operate directly on the native DOM elements):

```ts
const displayValue = selectBox.getDomElement().css('display');
selectBox.getDomElement().css('display', 'none');
...
selectBox.getDomElement().css('display', displayValue);
```